spec/models/program_spec.rb

```diff
@@ -90,8 +90,8 @@
   end
 
   describe 'voting_period?' do
-    it 'retuns false when voting dates are not set' do
-      expect(program.voting_period?).to eq false
+    it 'retuns true when voting dates are not set' do
```

Comment:
Compatibility with pre-existing conferences that do not have voting dates, and essentially no change in workflow for ppl already used to that logic. That practically means voting is still possible after the conference, which might not make much sense, but that's what happens now in our existing conferences.